plutus-tx-plugin/test/Budget/9.6/constAccR.uplc.golden

```diff
@@ -1,7 +1,9 @@
 (program
    1.1.0
-   ((\s -> s s)
-      (\s ds -> force (case ds [(delay 42), (\x xs -> delay (s s xs))]))
+   ((\x0 ->
+       (\s -> s s)
+         (\s ds -> force (case ds [(delay 42), (\x xs -> delay (s s xs))]))
+         x0)
       ((\s -> s s)
          (\s n ->
             force
```

Comment:
This I guess show the deficiency of the approach. It's nice that we can inline recursive functions better, but we still can't inline fully applied recursive functions.